docs/egg_model_example.rst

```diff
@@ -0,0 +1,149 @@
+
+::
+
+        COMPDAT
+            'INJECT1'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /                    
+            'INJECT2'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /            
+            'INJECT3'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT4'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT5'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT6'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT7'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'INJECT8'    2*    4     7 'OPEN' 2*     0.2 	1*          0 /
+            'PROD1'      2*    1     3 'OPEN' 2*     0.2 	1*          0 / 
+            'PROD2'      2*    1     3 'OPEN' 2*     0.2 	1*          0 / 
+            'PROD3'      2*    1     3 'OPEN' 2*     0.2 	1*          0 / 
+            'PROD4'      2*    1     3 'OPEN' 2*     0.2	1*          0 /
+        /
+        
+        
+
+2. The section PROPS is saved as an independent file `PROPS.inc`.  
+
+::
+
+            DENSITY
+                900 1000          1 /
+            PVCDO
+                400          1 1.000E-05        5        0/
+            PVTW
+                400          1 1.000E-05        1        0/
+
+                
+3. Output variables of interest have been specified in the SUMMARY section.
+
+::
+
+        SUMMARY
+            FOPR
+            FWPR
+            FWIR
+            WOPR
+            'PROD1'
+            'PROD2'
+            'PROD3'
+            'PROD4'
+            /
+            WWPR
+            'PROD1'
+            'PROD2'
+            'PROD3'
+            'PROD4'
+            /
+            WWIR
+            'INJECT1' 
+            'INJECT2'
+            'INJECT3' 
+            'INJECT4' 
+            'INJECT5'
+            'INJECT6'
+            'INJECT7'
+            'INJECT8'
+            /   
+            FOPT
+            FWPT
+            FWIT
+            WLPR
+            'PROD1'
+            'PROD2'
+            'PROD3'
+            'PROD4'
+            /
+            WBHPH
+            /
+            WBHP
+            /
+
+
+4. Then you can run OPM FLOW to update input files to FlowNet
+
+.. code-block:: bash
+
+    flow /egg_model/input_file/Egg_Model_ECL.DATA
+    
+    
+    
+
+
+Assisted history matching example
+=================================
+
+The configuration files follow the `YAML standard <https://yaml.org/>`_.
+
+.. literalinclude:: ../examples/egg_parameters.yml
+   :language: yaml
+   :linenos:
+   
+   
+To run the assisted history matching in flownet:
+
+
+.. code-block:: bash
+
+   flowet ahm /config_file_directory/egg_parameters.yml  output_directory
+   
+   
+
+Webiz visualization
+===================
+
+.. code-block:: bash
+
+    python output_directory/generated_app/webviz_app.py
+
+
+.. image:: ./_static/webviz_results_egg_model.png
```

Comment:
Here I suggest to put an inline webviz visualization, or something along those line - but you were already looking into that direction. Maybe we can't really automate that entire training for every new FlowNet release, so "static" files could be good enough. Up to you @anders-kiaer.